### ISSUE TITLE
No action when clicking a spot in SmartSDR for Mac

### ISSUE THREAD

**Issue by ct1drb** [tnxqso/wave-flex-integrator#15]
Hello all,

I am using a macOS computer and Wave-Flex Integrator is not working well concerning to add callsign into the QSO windon, more specifically at the Callsign field.
Looking at the terminal where I start the Wave-Flex Integrator there are 2 lines that are missing when I pick up a certain spot:
the spot triggered line and the Opened URL line.
It seems that without the triggered line there is not the Opened URL line.
is there anyone running the Wave-Flex Integrator in a macOS computer ?  

Best 73 all.

CT1DRB/OK8RB
David Quental

**Comment by zone11** [comment]
@ct1drb I'm trying to use it on macOS with SmartSDR from Marcus. Adding the spots to the pan adapter works fine here, but no action is performed when clicking on it. I'm not sure if this is supported.

**Comment by ct1drb** [comment]
Hello Cristian,

how are you doing?

When you hover a certain callsign from panadaptor can you check if date and time are there ?

In my case there is not date and time and I think it is the problem.

Best 73 es tks.

CT1DRB/OK8RB
David Quental

**Comment by zone11** [comment]
Hi David

This how a spot looks like, no date or time:
![Bildschirmfoto 2024-10-07 um 12 37 50](https://github.com/user-attachments/assets/ecab4b3b-01de-4111-b174-79602e9a7457)

73 de HB9HJQ, Christian

**Comment by besynnerlig** [comment]
@ct1drb, David has confirmed this behavior by connecting to his FlexRadio via telnet on port 4992. FlexRadio is not responding with a message similar to S5FB40D3F|spot 5934 triggered pan=0x40000000 when a spot on the SmartSDR for Mac panadapter is clicked (as it does on SmartSDR for Windows). I have reached out to HAM-Radio-Apps.com about this issue, but it may take some time to receive a response.

**Comment by besynnerlig** [comment]
I have received information today from Marcus, DL8MRE one of the developers of [SmartSDR for Mac](https://roskosch.de/smartsdr-for-mac/) and SmartSDR for iOS. The simple answer is that his app (SmartSDR for Mac) is not supposed to do that (Informing the Flexradio when a spot is clicked in the UI). So this is not a bug. I have updated the [README](https://github.com/tnxqso/wave-flex-integrator/blob/main/README.md) file with information about that when using SmartSDR for Mac, the Click-to-Log Feature is not supported due to software limitations. E.g. SmartSDR for Mac does not send information about clicked spots back to the FlexRadio, preventing Wave-Flex Integrator from triggering the logging action.

**Comment by zone11** [comment]
Well "is not supposed to do that" isn't the answer we were hoping for..

**Comment by besynnerlig** [comment]
> Well "is not supposed to do that" isn't the answer we were hoping for..

You are right about that. Maybe they haven't seen the need for it. Let's hope that they will consider adding that functionality if several customers are requesting it. To be able to open the logging window prefilled when clicking on a spot is a nice feature.

**Comment by zone11** [comment]
I'm going to send a request to Marcus, maybe @ct1drb you can also do so.

**Comment by ct1drb** [comment]
Hello Christian and all,

how are you doing?

> No dia 08/10/2024, às 15:29, Christian ***@***.***> escreveu:
> 
> 
> I'm going to send a request to Marcus, maybe @ct1drb <https://github.com/ct1drb> you can also do so.
> 

Yes Christian, I have done request to Marcus  already.


Best 73 es tks all.

CT1DRB/OK8RB
David Quental

**Comment by ok1cdj** [comment]
Hello all, do you have any feedback from Markus ?

Regards, Ondra OK1CDJ

**Comment by besynnerlig** [comment]
> Hello all, do you have any feedback from Markus ?
> 
> Regards, Ondra OK1CDJ

Hi Ondra,

I emailed Markus four days ago but haven’t received a reply yet. It’s possible he didn’t see it, or he may simply be busy. In the meantime, I plan to post a message on the [FlexRadioSmartSDR](https://groups.io/g/FlexRadioSmartSDR) discussion group about my migration to the Mac environment. Overall, the experience has been positive, though I really miss having clickable spots. I hope to share more details with the community soon.
